lib/widgets/pages/login/login_passcode_page.dart

```diff
@@ -21,7 +27,64 @@ class LoginPasscodePage extends LoginPage {
                 context.read<LoginCubit>().loginAsAnotherUser();
               },
             ),
+            const Gap(8),
```

Comment:
Now just remove this gap and enable **auto _squash_ and merge**, then everything looks good!

Reply:
you just said it should be there:
"
   *  The spacing between CTAs should be 8.
    * Preferably, the forgot button is in the passcode page as either a numpad action or a CTA, see picture
"

Reply:
My bad, I assumed the text button didn't have invisible padding, so I didn't take it into account. But without any Gap widget there, the two CTAs look just about the correct distance away from each other.